extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -157,15 +159,21 @@ The `services` property contains an array of service configurations. There must
 If you'd like to test the document's behavior in the context of a particular viewer, you can add `#viewerUrl=` fragment parameter. For instance, `#viewerUrl=https://www.google.com` would emulate a document's behavior inside a Google viewer.
 
 
-## Selecting platform
-So if no platforms are selected, we compete all the platforms based on platforms like
+## Selecting a Service
+No service returns an entitlement that grants access we compete all the service by calculating a score for each and choosing the highest scoring service.
 
-1. Does the platform support the Viewer
+The score is calculated by taking the `baseScore` in the service configuration and adding or subtracting dynamically calculated weights from `score[factor name]` if the service indicates that `factor name` is true for this page view.
 
-You can add `"baseScore"` < 100 key in any service configuration in case you want to increase `"baseScore"` of any platform so that it wins over other score evaluation factors.
+Available scoring factors:
+
+1. `supportsViewer` Indicates that a service can cooperate with the current AMP viewer environment for this page view.
+
+In addition to the dynamic scoring factors each service has a `"baseScore"` (default 0). A value < 100 in the `baseScore` key in any service configuration represents the initial score for that service.
+
+In the event of a tie the local service wins.
 
 ## Error fallback
-In case if all configured platforms fail to get the entitlements, the entitlement configured under `fallbackEntitlement` section will be used as a fallback entitlement for `local` platform. The document's unblocking will be based on this fallback entitlement.
+In case if all configured services fail to get the entitlements, the entitlement configured under `fallbackEntitlement` section will be used as a fallback entitlement for `local` service. The document's unblocking will be based on this fallback entitlement.
```

Comment:
Remove "In case":

If all configured services fail to get the entitlements, the entitlement configured under `fallbackEntitlement` section will be used as a fallback entitlement for `local` service.

Or update "if" to "that" (if we need to clarify specific case):

In the case that all configured services fail to get the entitlements, the entitlement configured under `fallbackEntitlement` section will be used as a fallback entitlement for `local` service.